pages/developers/resources/indexer.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Indexer
+sidebar_position: 3
+---
+
```

Comment:
```suggestion
```
You can remove this frontmatter. You'll have to update `/pages/developers/sidebars.yaml` to get the page on the sidebar in the developer section.

```
- group: Resources
  expanded: false
  items:
    - page: ./resources/indexer.md
    - page: ./resources/block-explorers.md
    - page: ./resources/public-apis.md
```